osgp/protocol-adapter-iec60870/osgp-protocol-adapter-iec60870/src/main/java/org/opensmartgridplatform/adapter/protocol/iec60870/application/mapping/Iec60870Mapper.java

```diff
@@ -17,12 +19,26 @@
 
 public class Iec60870Mapper extends ConfigurableMapper {
 
+  private final TimeZone timeZone;
+
+  public Iec60870Mapper(final TimeZone timeZone) {
+    super(false);
+
+    this.timeZone = timeZone;
+  }
+
+  @PostConstruct
+  public void initialize() {
+    this.init();
+  }
```

Comment:
Would this also work? 
```suggestion
  public Iec60870Mapper(final TimeZone timeZone) {
    super(false);
    this.timeZone = timeZone;
    this.init();
  }
```
If so, there would be no need to call the `initialize` method in `Iec60870AsduConverterTest` and other test classes.

Reply:
Seems to work fine, updated the code.